apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -138,12 +155,14 @@ open class ChatController(private val ctrl: ChatCtrl, val ntfManager: NtfManager
       if (chatStarted) {
         chatModel.chats.clear()
         chatModel.chats.addAll(chats)
+        chatModel.chatRunning.value = true
```

Comment:
I think it should be done in both branches - not sure if it's possible that this value is not set, but `chatStarted` is false only if the chat was already running, so this value must be true in both cases.